src/metabase/api/card.clj

```diff
@@ -166,15 +185,7 @@
   (let [raw-card (t2/select-one Card :id id)
         card (-> raw-card
                  api/read-check
-                 (t2/hydrate :creator
-                             :dashboard_count
-                             :parameter_usage_count
-                             :can_write
-                             :average_query_time
-                             :last_query_start
-                             [:collection :is_personal]
-                             [:moderation_reviews :moderator_details])
-                 collection.root/hydrate-root-collection
+                 hydrate-card-details
                  (cond-> ;; card
                    (:dataset raw-card) (t2/hydrate :persisted))
```

Comment:
Just spotted you don't need this anymore. You're doing it in `hydrate-card-details`.
```suggestion
```